README.md

```diff
@@ -160,7 +160,7 @@ To learn more about this, see the [`optimade-python-tools`](https://github.com/M
 
 ## For maintainers
 
-To release the new version, go to GitHub release API of the repo create a new release and update the release information. 
+To release the new version, go to GitHub release API of the repo create a new release and update the release information.  
```

Comment:
I think this is a case of where the trailing-whitespace hook correctly removed the trailing white space that wasn't meant to be there. I.e., I don't think a newline was intended here. But if you prefer it, please keep it, of course.
```suggestion
To release the new version, go to GitHub release API of the repo create a new release and update the release information.
```